### ISSUE TITLE
[1.12.2] No proper name or description

### ISSUE THREAD

**Issue by JoeSGT** [MachineMuse/MachineMusePowersuits#870]
Here what it is https://gyazo.com/bd0e9d05aec4067b026072a4e84a80f6
It's on the Power fist and it's missing a name and description on something i took a screenshot it's a small bug but it's worth reporting :)

**Comment by lehjr** [comment]
Yeah, that's one of the modules still under development. Unfortunately, there's still a bunch of stuff left to translate, including all of the config settings.
